src/main/java/avenga/random/service/SheetsService.java

```diff
@@ -0,0 +1,66 @@
+package avenga.random.service;
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential;
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.jackson2.JacksonFactory;
+import com.google.api.services.sheets.v4.Sheets;
+import com.google.api.services.sheets.v4.SheetsScopes;
+import com.google.api.services.sheets.v4.model.ValueRange;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import javax.annotation.PostConstruct;
+import java.io.IOException;
+import java.security.GeneralSecurityException;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+@Service
+public class SheetsService {
+    public static final String SERVICE_CREDENTIALS = "/service-keys.json";
```

Comment:
credentials should be served as via application.properties and would be stored in Heroku as ENV_VARIABNLES. For more info on that plz ask Mykhailo